src/slurm_plugin/clustermgtd.py

```diff
@@ -1126,36 +885,123 @@ def _handle_protected_mode_process(self, slurm_nodes, partitions, bootstrap_fail
         # Partition does not have job running
         log.info(
```

Comment:
We don't need to print out this log in there is no failure count? This way we can reduce some clutter in the logs?